### PR TITLE
Created Bunny Ears accessory

### PR DESCRIPTION
**Pull request by jakerockland**
##### Description

Adds bunny ear accessory to avatar creator, re: #2333

##### Checklist

- [x] linter status: 100% pass
- [x] changes don't break existing behavior
- [x] commit message follows [commit guidelines](https://github.com/gitcoinco/web/blob/master/docs/CONTRIBUTING.md#step-4-commit)

##### Affected core subsystem(s)

avatar

##### Testing

Tested locally, screenshots attached:

screenshot 2018-10-19 01 35 12
screenshot 2018-10-19 01 36 26

##### Refers/Fixes

Refs: #2333

### CODE REVIEW

**Comment by jakerockland** [comment]
@thelostone-mc So I think I fixed things, but there is a CodeCov failure now, is that something I need to pay mind to for this PR?

**Comment by jakerockland** [comment]
Fixed merge conflict 👍

File: app/avatar/utils.py [hunk 1]
```diff
@@ -101,8 +101,9 @@ def get_avatar_context():
                                      ['HatShort-redbow'], ['HatShort-yellowbow'],
                                      ['HatShort-ballcap'], ['HatShort-cowboy'],
                                      ['HatShort-headphones'], ['HatShort-shortbeanie'],
-                                     ['HatShort-tallbeanie'], ['Earring-0'],
-                                     ['Earring-1'], ['EarringBack-2', 'Earring-2'],
+                                     ['HatShort-tallbeanie'], ['HatShort-bunnyears'],
+                                     ['Earring-0'], ['Earring-1'],
```
[the remark this answers is not in the document]
Comment (thelostone-mc) [reply app/avatar/utils.py]: @jakerockland could you address this too?
And the CodeCov failure we can ignore for this PR

**Comment by jakerockland** [comment]
Ah rad thank you for doing that @thelostone-mc

**Pull request merged**